debian/etc/apache2/sites-available/nextcloud.conf

```diff
@@ -0,0 +1,8 @@
+Alias /nextcloud /usr/share/nextcloud/
+<Directory "/usr/share/nextcloud/">
+Options MultiViews FollowSymlinks
```

Comment:
`MultiViews` shouldn't be necessary?

Reply:
I'm not sure, how can i test it

Reply:
In ownCloud 8.1, it was [explicitly forbidden](https://doc.owncloud.org/server/8.1/admin_manual/release_notes.html#disable-apache-multiviews). I can't imagine that they started depending on it in the meantime.